TheMovieDb/Models/MovieDetail.swift

```diff
@@ -10,12 +10,25 @@ import Foundation
 struct MovieDetail: Codable {
     let poster_path: String?
     let overview: String
-    var items: [String] = ["", ""]
+    let title: String
+    let year: String
+    let rate: String
+    var items: [String] = ["", "", "", "", ""]
```

Comment:
Yep, it is not used because of the SwiftUI implementation, and I left it in case I may have to swap the method which fills the movie detail. I'm going to remove it then.